### ISSUE TITLE
RSS Feed needs styling for metadata

### ISSUE THREAD

**Issue by bph** [WordPress/wporg-developer-blog#30]
"Text could use the same font size, and post meta could be unified on a single row, like /news:". 
![Screen Shot 2022-11-30 at 15 11 54](https://user-images.githubusercontent.com/39980/204898592-a6b92bda-90a8-419a-b65e-8911828c73e8.png)

Date and author should be on one line underneath the post title

**Comment by ryelle** [comment]
I don't think we can alter the RSS block, but we might be able to update [the `latest-news` block](https://github.com/WordPress/wporg-mu-plugins/tree/trunk/mu-plugins/blocks/latest-news) to allow selecting a different WordPress.org blog.

That block is what powers this on the homepage
<img width="1252" alt="Screenshot 2023-01-06 at 12 23 10 PM" src="https://user-images.githubusercontent.com/541093/211064638-f5e4dbb3-0a84-4f93-866b-efd5d0b7735d.png">

**Comment by bph** [comment]
That would be a great alternative, for sure!

- needs to allow for selecting a certain tag on the blog, here the dev-notes as in https://make.wordpress.org/core/tag/dev-notes/
- needs to be possible to have the same block twice on the page (Sometimes is not possible with dynamic blocks)

**Comment by ryelle** [comment]
> needs to allow for selecting a certain tag on the blog

That's not a feature of this block right now, but it could be added.

> needs to be possible to have the same block twice on the page

I believe this is fine right now, but where else will this be needed?

**Comment by bph** [comment]
Fabulous. It probably will take a while until the block is revamped to use for other WordPress blogs. 

On the homepage the block powers the list of the latest updates. 
If it's also used to replace the RSS feed, then there will be another instance of the block

The Make Core blog's feed is inherently too noisy for outside developers. The only important posts on that make blog are the dev notes, so the list would need to show only those.

**Comment by ryelle** [comment]
>  If it's also used to replace the RSS feed, then there will be another instance of the block

Where else would it be used? This block won't necessarily replace _any_ RSS feed, it needs to be a site on the wporg network.

**Comment by bph** [comment]
One the homepage we use one Latest post blog for the list of posts on this site. 
And then an RSS block for thelist of Dev Notes from the Make Blog. If I understood the proposed changes correctly we would after the revamp use two Latest Posts Blocks on the front page:

For pulling the post from this site, and one pulling the posts from the Make block tag 'devnotes' now pulled in by the RSS Block. 

>This block won't necessarily replace any RSS feed, it needs to be a site on the wporg network.
I understood, but it needs the feature to pull from the tag feed if we use it for the frontpage to replace the RSS feed there.

**Comment by ryelle** [comment]
> One the homepage we use one Latest post blog for the list of posts on this site.

Oh! That's just the regular Query Loop block, we don't need to do anything fancy for that :) Unless it's not working the way you want it to?

> And then an RSS block for thelist of Dev Notes from the Make Blog.

Right, this will use the custom Latest News block. It's been updated to pull from any site on WordPress.org, but it does still need an update to filter by taxonomy so that we can use it for just `dev-notes`.

**Comment by bph** [comment]
Is there a meta trac ticket that I could follow?

**Comment by ryelle** [comment]
I added an issue here: https://github.com/WordPress/wporg-mu-plugins/issues/324
